src/index.tsx

```diff
@@ -113,24 +136,15 @@ export const Editor: React.FunctionComponent<DemoProps> = ({
     }
   }, [height, value, setHeight]);
 
-  const { handleSuggestionSelected, handleKeyDown, handleKeyPress } =
-    getHandlers({
-      ref,
-      suggestions,
-      setShowSuggestion,
-      showSuggestion,
-      setFocusIndex,
-      focusIndex,
-      setCaret
-    });
-
-  const isEditingText = React.useMemo(() => {
-    if (showSuggestion) {
-      return false;
-    }
-
-    return true;
-  }, [showSuggestion]);
```

Comment:
line 153 needs this value #58

Reply:
this purpose to: if current input status in some features(e.g. mention), will not execute the `newline` command.

Reply:
restored